covid_graphs/covid_graphs/formula.py

```diff
@@ -95,26 +110,25 @@ def get_curve(self, country_report: CountryReport) -> Curve:
         fit = fit_atg_model.fit_atg_model(
             xs=xs, ys=country_report.cumulative_active[: until_idx + 1],
         )
+        prefix = f"{self.until_date.strftime('%b %d')}: "
+        label = create_label(fit.a, fit.tg, fit.exp, prefix=prefix)
         # TODO(miskosz): Remove the print when we integrate with a dashboard.
         print(fit)
 
         start_date = date_zero + datetime.timedelta(days=fit.t0)
-        end_date = _get_display_end_date(
-            tg=fit.tg, exp=fit.exp, start_date=start_date, data_end_date=country_report.dates[-1],
+        display_at_least_until = _get_display_at_least_until(
+            tg=fit.tg, exp=fit.exp, start_date=start_date,
         )
 
-        def formula(date: datetime.date):
-            x = (date - date_zero).days
-            return fit.predict(x)
-
-        return Curve(func=formula, start_date=start_date, end_date=end_date, label=label)
+        return Curve(
+            func=lambda x: fit.predict(x),
```

Comment:
Re: Logic leaking in `fit_atg_model`: Now we are passing different `x` in then we were before. We should keep it consistent. Also we should be careful, `fit.t0` can be non-integral.